### PR TITLE
EVG-12757: task queue table

### PR DESCRIPTION
**Pull request by tgrander**
Basic task queue table

Screen Shot 2020-08-24 at 1 08 07 PM

### CODE REVIEW

File: src/pages/TaskQueue.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+} from "gql/generated/types";
+import { DISTRO_TASK_QUEUE } from "gql/queries";
+import { TableContainer, PageWrapper } from "components/styles";
+import { Body } from "@leafygreen-ui/typography";
+
+export const TaskQueue = () => {
+  const { data: taskQueueItemsData, loading } = useQuery<
+    DistroTaskQueueQuery,
+    DistroTaskQueueQueryVariables
+  >(DISTRO_TASK_QUEUE, { variables: { distroId: "rhel71-power8-small" } });
```
Comment (tgrander) [inline src/pages/TaskQueue.tsx:18]: this is meant to be static for now. will update based on url param in future pr.

Comment (SupaJoon) [reply src/pages/TaskQueue.tsx:18]: will this be the default param in the URL?

Comment (tgrander) [reply src/pages/TaskQueue.tsx:18]: no

**Comment by tgrander** [comment]
> Can we add a test to verify that the table renders with some data?

this is the very first PR of several. i will add tests as i add features.

**Pull request merged**